### PR TITLE
Enable npm packaging, including .css and .scss

### PR DESCRIPTION
**Pull request by mbland**
Per the discussion on #925. Still needs documentation regarding how to use the `css` and `scss` keys. For reference, here's some source material for the forthcoming documentation:

- http://stackoverflow.com/questions/32037150/style-field-in-package-json
- http://techwraith.com/your-css-needs-a-dependency-graph-too/
- https://www.npmjs.com/package/rework-npm
- https://www.npmjs.com/package/resin
- https://www.npmjs.com/package/parcelify

cc: @maya @msecret @jeremiak

### CODE REVIEW

**Comment by jeremiak** [comment]
:+1: 

I didn't know about `parcelify` before, nice find @mbland!

File: package.json [hunk 1]
```diff
@@ -2,13 +2,14 @@
   "name": "uswds",
   "version": "0.9.0",
   "description": "Open source UI components and visual style guide for U.S. government websites",
-  "main": "package.json",
+  "main": "assets/js/components.js",
   "scripts": {
     "build-sass": "node-sass assets/_scss/all.scss > assets/css/uswds.css",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "prepublish": "npm run build-sass build"
```
Comment (msecret) [inline package.json]: Isn't it just supposed to be `npm run build-sass`?

**Comment by msecret** [comment]
Why wouldn't we use a `style` property as well as `css` and `scss`?

File: package.json (hunk 1 above)
Comment (mbland) [inline package.json]: Good catch. I'd had a separate "build" target previously that I'd since taken out. Done.

**Comment by mbland** [comment]
@msecret Added a `style` key. PTAL.

**Comment by maya** [comment]
Does this PR need to be opened against `v0.9.x` rather than `18f-pages-staging`?

**Comment by msecret** [comment]
LGTM

**Comment by maya** [comment]
Chatted with @msecret, we can cherry-pick the commits ourselves into `v0.9.x`. Thanks @mbland!

**Comment by maya** [comment]
Added into our `v0.9.x` release branch: https://github.com/18F/web-design-standards/tree/v0.9.x - thanks @mbland!

**Pull request closed without merging**